src/templates/home-page/Hero/Analytics.tsx

```diff
@@ -0,0 +1,45 @@
+import { Stack, Button, StackProps } from "@chakra-ui/react"
+import { FC } from "react"
+import { StatBoxGroup } from "../../../components/StatBox"
+import { formatFiatCurrencyAmount, formatTokenAmount } from "../../../utils"
+
+interface AnalyticsProps {
+  tbtcTtvl: string // in wei
+  tbtcUniqueAddresses: string
+  stakingTvlInUSD: string // without the `$` sign
+}
+
+export const Analytics: FC<AnalyticsProps & StackProps> = ({
+  tbtcTtvl,
+  tbtcUniqueAddresses,
+  stakingTvlInUSD,
+  ...restProps
+}) => {
+  const stats = [
+    {
+      // TODOL make the `btc` word smaller
+      amount: `${formatTokenAmount(tbtcTtvl, "0,00.00")}btc`,
+      label: "tBTC TVL",
+      variant: "secondary",
+    },
+    {
+      amount: tbtcUniqueAddresses,
+      label: "tBTC unique address",
+      variant: "secondary",
+    },
+    {
+      amount: formatFiatCurrencyAmount(stakingTvlInUSD, "0a").toUpperCase(),
```

Comment:
What does `0a` mean?

Reply:
It cuts the thousands, millions etc. and replace it with a proper letter. For example:
1 000 -> 1k
20 000 000 -> 20m

See http://numeraljs.com/